File: Project/Frontend/daariiou-photography-frontend/src/api/lib/models/shooting.ts
/* tslint:disable */
/* eslint-disable */
import { KindOfShooting } from './kind-of-shooting';
import { User } from './user';
export interface Shooting {
  date?: string;
  kos?: null | KindOfShooting;
  kosId?: number;
  reasonDeclined?: null | string;
  remarks?: null | string;
  sid?: number;
  status?: null | string;
  uid?: number;
  uidNavigation?: null | User;
}

File: Project/Backend/daariiou_photography_backend/daariiou_photography_backend/Models/Shooting.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace daariiou_photography_backend.Models
{
    public partial class Shooting
    {
        public int Sid { get; set; }
        public DateTime Date { get; set; }
        public string Remarks { get; set; }
        public string ReasonDeclined { get; set; }
        public string Status { get; set; }
        public int Uid { get; set; }
        public int KosId { get; set; }

        public virtual KindOfShooting Kos { get; set; }
        public virtual User UidNavigation { get; set; }
    }
}
